windows-iot/iot-enterprise/Soft-Real-Time/Soft-Real-Time-Device.md

```diff
@@ -147,9 +147,8 @@ Environments that use Windows Management Instrumentation (WMI) can use the MDM B
   Set-CimInstance -CimInstance $obj
 
   ```
-
 >[!TIP]
 >
 > You can use the same script for whatever number of real-time cores you need to have, just replacing the 3 in the second-to-last line with the appropriate number. This will reserve cores starting with core 0 and going upwards. So reserving 3 cores on a 4 core CPU will reserve cores 0, 1, and 2 and leave core 3 for system and non-real-time tasks.
-
+# How to check the Device  Real-Time Performance Working
```

Comment:
What is the purpose of adding this Heading 1 in this location of the document without content to support it?